fhir-persistence/src/main/java/org/linuxforhealth/fhir/persistence/index/SearchParametersTransportAdapter.java

```diff
@@ -53,15 +63,17 @@ public SearchParametersTransport build() {
     }
 
     @Override
-    public void stringValue(String name, String valueString, Integer compositeId, boolean wholeSystem) {
+    public void stringValue(String name, String valueString, Integer compositeId, boolean wholeSystem, int maxBytes){
```

Comment:
super minor style thing
```suggestion
    public void stringValue(String name, String valueString, Integer compositeId, boolean wholeSystem, int maxBytes) {
```

Reply:
This has been fixed as suggested.